lib/Runtime/Base/WindowsGlobalizationAdapter.cpp

```diff
@@ -829,20 +829,30 @@ if (this->object) \
         return ResolveLocaleLookup(scriptContext, locale, resolved);
     }
 
-    int IcuIntlAdapter::GetUserDefaultLocaleName(_Out_ LPWSTR lpLocaleName, _In_ int cchLocaleName)
+    size_t IcuIntlAdapter::GetUserDefaultLanguageTag(_Out_ char16* langtag, _In_ size_t cchLangtag)
     {
-        // XPLAT-TODO (doilij): implement GetUserDefaultLocaleName
-        const auto locale = _u("en-US");
-        const size_t len = 5;
-        if (lpLocaleName)
+        UErrorCode error = U_ZERO_ERROR;
```

Comment:
Use `UErrorCode::U_ZERO_ERROR` to be explicit, and consistent with other places (I've made this consistent in my PR as well).

Reply:
Wouldn't that be using a C++ enum class? Are we still going forward with adding more C++ until we resolve the system ICU issue?

Reply:
Doesn't require an enum class but in C++ all enum values are namespace-scoped to their enclosing enum. I'm not sure we can resolve the system ICU issue (might not be possible and we just have to build it in any case) but for now we rely on the C++ APIs, so IMO we should be consistent.